MushroomObserver/mushroom-observer
# Versioned enums

Uses a patched version of `acts_as_versioned` which stores enum attributes as integers in Version tables

Delivers https://www.pivotaltracker.com/story/show/163189614

### Suggested Manual Test
* Create Name - Rank: `Genus`, `Candy Cap` 
* Edit Name to be Rank: `Species`, Scientific Name: `Candy cap Mushroom` 
* Edit Name to be Rank: `Subspecies`, Scientific Name: `Candy cap subsp. maple` Author: `DeShazer`
* Click `Previous Version`
Expected Result: Displays "Version 2: **_Candy cap_** Mushroom" with `Rank: Species`
* Click `Version 1`
Expected Result: Displays "Version 1: **_Candy_** Cap" with `Rank: Genus`
